### PR TITLE
REV: NOS-3 Enable Offliner to read <checksum>:<path> manifest path li…

### PR DESCRIPTION
**Pull request by yma88**
Enable Offliner to read  checksum : path manifest path listings, validate checksums

### CODE REVIEW

**Comment by jdcasey** [comment]
add to whitelist

File: src/main/java/com/redhat/rcm/offliner/alist/PlaintextArtifactListReader.java [hunk 1]
```diff
@@ -35,11 +36,15 @@
 public class PlaintextArtifactListReader implements ArtifactListReader
 {
 
+    public PlaintextArtifactListReader()
+    {
+    }
+
```
Comment (jdcasey) [inline src/main/java/com/redhat/rcm/offliner/alist/PlaintextArtifactListReader.java]: not strictly needed if it's the only constructor. Just so you know

**Comment by jdcasey** [comment]
retest this please

File: src/main/java/com/redhat/rcm/offliner/alist/PlaintextArtifactListReader.java (hunk 1 above)
Comment (yma88) [inline src/main/java/com/redhat/rcm/offliner/alist/PlaintextArtifactListReader.java]: oops, yeah, will delete this in next commit

**Comment by yma88** [comment]
John, how is the maven test running result? I runs well in local, what's the matter for retest?

**Comment by jdcasey** [comment]
@yma88 Sorry, those comments are actually directed at the CI server...it monitors PRs for that text, and is supposed to fire off a CI build when it sees that. I was thinking that there was some change since the last CI build, but there wasn't. BTW if you want to see the CI build, you can click on the green check mark above (or the yellow dot if the build is in progress...or the red x if the build failed).

**Pull request merged**